system-addon/content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -152,9 +152,11 @@ export class _CollapsibleSection extends React.PureComponent {
     const disclaimerPref = `section.${id}.showDisclaimer`;
     const needsDisclaimer = disclaimer && this.props.Prefs.values[disclaimerPref];
     const active = menuButtonHover || showContextMenu;
-
     return (
-      <section className={`collapsible-section ${this.props.className}${enableAnimation ? " animation-enabled" : ""}${collapsed ? " collapsed" : ""}${active ? " active" : ""}`}>
+      <section
+        className={`collapsible-section ${this.props.className}${enableAnimation ? " animation-enabled" : ""}${collapsed ? " collapsed" : ""}${active ? " active" : ""}`}
+        // Note: data-section-id is used for web extension api tests in mozilla central
+        data-section-id={id}>
```

Comment:
Pretty sure data attribute values can have any string value, so we don't need to additionally clean up / sanitize… yeah?

Reply:
yes, rendering the id as data attribute should always be a string, so it shouldn't be a problem here